src/App.tsx

```diff
@@ -18,11 +21,24 @@ import {
 } from "./screens";
 import colors from "./utils/colors";
 import sizes from "./utils/sizes";
+import * as localStorage from "./services/localStorage";
 
 YellowBox.ignoreWarnings(["AsyncStorage has been extracted"]);
 
 const Stack = createStackNavigator();
 
+const addNewProcedure = async (navigation: StackNavigationProp<any, any>) => {
+  const { technology } = await localStorage.getPreferences();
+  const { role } = await localStorage.getPreferences();
```

Comment:
Você pode economizar uma leitura do AsyncStorage:

```suggestion
  const { technology, role } = await localStorage.getPreferences();
```

Reply:
Mudança feita.